code/modules/admin/verbs/playsound.dm

```diff
@@ -67,7 +67,7 @@
 	if(istext(web_sound_input))
 		var/web_sound_url = ""
 		var/stop_web_sounds = FALSE
-		var/pitch
+		var/list/music_extra_data = list("pitch", "seek")

```

Comment:
Just start it off as an empty list, you can assign to nonexistent keys without issue and trying to access those same keys will return null just fine

Reply:
Thanks, I am already aware of this. The intent is to let people know what keys are there to be used easily. With that in mind, do you still want it to be an empty list?

Reply:
This issue is still valid in my most recent commit, I simply added "end" to the list.